the-devastating-chickens/BangazonSite
# Adding Seed data to ApplicationDbContext.cs and initial migration

# Description

Creating additional instances of resources in ApplicationDataContex.cs. Creating initial migration and seeding the data in the db.

Fixes Issue # (n/a)

## Type of change


# Testing Instructions

1. `git fetch --all`
1. `git checkout initial-setup`
1. Select Tool => NuGet Package Manager => Package Manager Console.
1. Execute `Update-Database` to update tables
1. Use the SQL Server Object to verify that everything worked as expected.

# Checklist:
